rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -178,12 +177,12 @@ def get_one_hot_encoded_labels(self, array, device):
         if device == "cpu":
             if (isinstance(array, np.ndarray)):
                 b.getOneHotEncodedLabels(self._handle, array.ctypes.data_as(ctypes.c_void_p), self._num_classes, 0)
-            elif (isinstance(array, torch.Tensor)):
+            else: # torch tensor
```

Comment:
How this is handled for Tensor Flow?

Reply:
For tensorflow, we use numpy and cupy for CPU and GPU backends